substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateOptions.java

```diff
@@ -425,6 +425,11 @@ public static boolean useLLVMBackend() {
     @Option(help = "Show native-toolchain information and image-build settings", type = User)//
     public static final HostedOptionKey<Boolean> DumpTargetInfo = new HostedOptionKey<>(false);
 
+    @Platforms({Platform.LINUX.class, Platform.DARWIN.class})
```

Comment:
Having a platform specific API option complicates all sorts of build scripts, so I think we should avoid that in general.

Reply:
I agree. Also providing proper behaviour for Windows will make this use of `@Platforms` obsolete. See https://github.com/oracle/graal/pull/2393#discussion_r415622948